static.config.js

```diff
@@ -340,7 +341,7 @@ const getOfficialDocumentCollectionData = async (page, instance, client) => {
     offeredBy: getOfferedByFromDepartments(officialDocumentCollection.departments),
   };
 
-  officialDocumentCollection.documents = await getOfficialDocumentCollectionDocuments(officialDocumentCollection.id, client);
+  officialDocumentCollection.lowerBound = await getOfficialDocumentCollectionLowerBound(officialDocumentCollection.id, client);
```

Comment:
We no longer need to fetch all of the documents (because we'll query them a page at a time via our site_search view). But at this step we can get our lowerBound, the earliest dated official_document in our collection. Maybe I'll get the total document count here as well.